docs/_pages/0020-preface.md

```diff
@@ -11,9 +11,9 @@
 
 title: Preface
 layout: post
-next_url: /docs/about/
-prev_url: /docs/community/
+prev_url: /docs/roadmap/
 permalink: /docs/navigation/
+next_url: /docs/about/
```

Comment:
Also +1 for the consistent `prevurl => permalink => nexturl` structure. 

it will help diff/merging and copy/pasting less error prone when/if we create more such pages.